src/components/List/ListItem.tsx

```diff
@@ -47,25 +56,33 @@ class ListItem extends UIComponent<Extendable<IListItemProps>, any> {
     /** Toggle debug mode */
     debug: PropTypes.bool,
 
-    header: PropTypes.any,
+    /** A list item can have an end media which appears at the rightmost end and only when the list item is hovered. */
```

Comment:
we shouldn't be specific about where it will appear - actually, themes will introduce ability to customize the layout of the component in pretty flexible way, so just the following would be enough. We could improve later

> End media.

Reply:
Ah, I see. Will keep this in mind next time. 👍